Subscriptions/src/grpc/subscription_operations.py

```diff
@@ -45,17 +45,17 @@ async def create_subscription(session: AsyncSession, email: str, subscription_ty
         ).scalars().first()
 
         if subscription is not None:
-            return CreateSubscriptionResponse(message="\nSubscription with this email already exists.")
+            return CreateSubscriptionResponse(message="Subscription with this email already exists.")
```

Comment:
This try;except has too many things inside itself. Please have as little as possible expressions per try;except block